docs/human.md

```diff
@@ -0,0 +1,30 @@
+# Visualizing human model in meshcat via SMPL-X
+
+## Installation
+
+The SMPL-X library and model data are not included in a standard RoboMeshCat packages.
+It must be installed as described in [SMPL-X package](https://github.com/vchoutas/smplx), i.e. by running
+
+```bash
+pip install smplx[all]
+```
+
+and then downloading the data from [here](https://smpl-x.is.tue.mpg.de/).
```

Comment:
maybe specify that 'model' data? Because there are a lot of downloads on the page